src/components/modeler/Modeler.vue

```diff
@@ -320,6 +320,16 @@ export default {
     highlightNode(node) {
       store.commit('highlightNode', node);
     },
+    blurFocusedScreenBuilderElement() {
+      const elementsToBlur = ['INPUT', 'SELECT'];
+      const shouldBlurActiveElement = elementsToBlur.some((element) => {
+        return document.activeElement && document.activeElement.tagName.includes(element);
```

Comment:
I was wondering if `document.activeElement.tagName` would ever be lowercase, but it seems like it isn't supposed to be.

Reply:
Yeah seems not to, but you never know with, say, Microsoft... should we convert the case to be safe you think?

Reply:
I don't think we need to worry about it; we're really only supporting Chromium (Chrome, new Edge), Safari and FF.

I just tested on latest Chrome, Safari, and FF on my mac, and also on Edge 18 and Edge 79 beta on Windows 10 in BrowserStack and it's returning uppercase strings in all cases.

The WHATWG DOM standard [mentions](https://dom.spec.whatwg.org/#dom-element-tagname):
> The tagName attribute’s getter must return this’s HTML-uppercased qualified name.

And the W3C DOM 4.1 spec [says](https://www.w3.org/TR/dom41/#dom-element-tagname):
> If the context object is in the HTML namespace and its node document is an HTML document, let qualifiedName be converted to ASCII uppercase.

Reply:
Good, let's keep to the standard